README.md

```diff
@@ -1,49 +1,47 @@
 # L[ightningPayJ]oin: Automated External Lightning Channel Funding
 
-This server leverages Pay-to-Endpoint to negotiate channel funding on your lightning node from an external wallet supporting BIP78. Previously, external funding could only be done by [signing PSBTs manually](https://github.com/lightningnetwork/lnd/blob/master/docs/psbt.md). A node running this software can open a new scheduled lightning channel upon payment receipt from e.g. BTCPayServer with a normal QR scan transfer flow.
+Loin leverages Pay-to-Endpoint (payjoin) to negotiate a channel open for your lightning node, initiated by any BIP78 supporting wallet. Previously, custom PSBT channel funding could only be done by [signing PSBTs manually](https://github.com/lightningnetwork/lnd/blob/master/docs/psbt.md). A node running Loin can open a new lightning channel for every received payment, provided the payer supports bip78. E.g. BTCPayServer or Blue wallet, following the normal QR scan transfer flow. Read the article about [Lightning Powered PayJoin](https://chaincase.app/words/lightning-payjoin?ref=github) to hear all the benefits!
```

Comment:
```suggestion
Loin leverages Pay-to-Endpoint ([BIP78](https://github.com/bitcoin/bips/blob/master/bip-0078.mediawiki) PayJoin) to negotiate channel establishment on your lightning node from any [supporting wallet](https://en.bitcoin.it/wiki/PayJoin_adoption). Previously, custom PSBT channel funding could only be done by [signing PSBTs manually](https://github.com/lightningnetwork/lnd/blob/master/docs/psbt.md). A node running Nolooking can open many lightning channels for every inbound transaction, provided the payer supports BIP78. E.g. BTCPayServer or Wasabi by following the normal QR scan transfer flow. Read the article about [Lightning Powered PayJoin](https://chaincase.app/words/lightning-payjoin?ref=github) to hear how it can save you time and money.
```

- Channel Establishment is the lingo the use in the [Bolt 02 spec](https://github.com/lightning/bolts/blob/master/02-peer-protocol.md#channel-establishment)
- stay consistent with bippersz
- BlueWallet not tested yet, nor JM. Is it? Wasabi is. Happy to add as soon as it's tested

Reply:
Ohh this is a tricky one. My reaction just then was that I had never heard of channel establishment. Do we want to go legit or colloquial? I am leaning towards "Channel open" or "channel funding" because that is familiar phrasing to users

Reply:
Familiar is OK with me. would just be good to remove Blue because that does not work as teested